chainer/functions/activation/softplus.py

```diff
@@ -58,18 +58,27 @@ def backward_gpu(self, inputs, grads):
 
 
 def softplus(x, beta=1.0):
-    """Elementwise softplus function.
+    """Element-wise softplus function.
 
-    This function is expressed as
-    :math:`f(x) = \\frac{1}{\\beta}\\log(1 + \\exp(\\beta x))`,
-    where :math:`\\beta` is a parameter.
+    .. math:: f(x)=\\frac{1}{\\beta}\\log(1 + \\exp(\\beta x)).
 
```

Comment:
Please add a brief explanation of this function here. SoftPlus is a smooth approximation to the ReLU, so the output value is always positive. And how the output value will change when the `beta` changed?